Content.Server.Database/ModelPostgres.cs

```diff
@@ -1,18 +1,33 @@
 ﻿using System;
+using System.Linq;
 using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Design;
 using Microsoft.EntityFrameworkCore.Diagnostics;
 using Microsoft.EntityFrameworkCore.Infrastructure;
 using Microsoft.EntityFrameworkCore.Storage;
+using NpgsqlTypes;
 
 namespace Content.Server.Database
 {
-    public sealed class PostgresServerDbContext : ServerDbContext
+    public sealed class PostgresServerDbContext : ServerDbContext, IDesignTimeDbContextFactory<PostgresServerDbContext>
     {
         // This is used by the "dotnet ef" CLI tool.
         public PostgresServerDbContext()
         {
         }
 
+        public PostgresServerDbContext(DbContextOptions<PostgresServerDbContext> options) : base(options)
+        {
+            Database.SetCommandTimeout(7200);
```

Comment:
I didn't put this in my initial review but this is a terrible idea. If the migration is a problem for this because it has to re-index, I should run it manually before merging the PR.

How heavy is this migration gonna be for us, and are there any potential problems with a sudden spike in disk usage or DB unavailability while it is happening?